libr/io/p/io_self.c

```diff
@@ -134,16 +134,16 @@ static int update_self_regions(RIO *io, int pid) {
 #ifdef _MSC_VER
 	int perm;
 	const size_t name_size = 1024;
-	ut64 to = 0;
+	PVOID to = NULL;
 	MEMORY_BASIC_INFORMATION mbi;
 	HANDLE h = OpenProcess (PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, 0, pid);
-	char *name = calloc (name_size, sizeof (char));
+	LPTSTR name = calloc (name_size, sizeof (char));
```

Comment:
Is it ok to use `sizeof(char)` for Unicode?

Reply:
Fixed